ingeniamotion/configuration.py

```diff
@@ -184,7 +189,80 @@ def set_max_acceleration(self, acceleration, servo=DEFAULT_SERVO,
         self.logger.debug("Max acceleration set to %s", acceleration,
                           axis=axis, drive=self.mc.servo_name(servo))
 
-    def set_max_velocity(self, velocity, servo=DEFAULT_SERVO, axis=DEFAULT_AXIS):
+    def set_max_profile_acceleration(self, acceleration, servo=DEFAULT_SERVO,
+                                     axis=DEFAULT_AXIS):
+        """Update maximum profile acceleration register.
+
+        Args:
+            acceleration: maximum profile acceleration in rev/s^2.
+            servo (str): servo alias to reference it. ``default`` by default.
+            axis (int): servo axis. ``1`` by default.
+        """
+        self.mc.communication.set_register(
+            self.PROFILE_MAX_ACCELERATION_REGISTER,
+            acceleration,
+            servo=servo,
+            axis=axis
+        )
+        self.logger.debug("Max profile acceleration set to %s", acceleration,
+                          axis=axis, drive=self.mc.servo_name(servo))
+
+    def set_max_profile_deceleration(self, deceleration, servo=DEFAULT_SERVO,
+                                     axis=DEFAULT_AXIS):
+        """Update maximum profile deceleration register.
+
+        Args:
+            deceleration: maximum profile deceleration in rev/s^2.
+            servo (str): servo alias to reference it. ``default`` by default.
+            axis (int): servo axis. ``1`` by default.
+        """
+        self.mc.communication.set_register(
+            self.PROFILE_MAX_DECELERATION_REGISTER,
+            deceleration,
+            servo=servo,
+            axis=axis
+        )
+        self.logger.debug("Max profile deceleration set to %s", deceleration,
+                          axis=axis, drive=self.mc.servo_name(servo))
+
+    def set_profiler(self, acceleration=None, deceleration=None, velocity=None,
+                     servo=DEFAULT_SERVO, axis=DEFAULT_AXIS):
+        """Update maximum acceleration and deceleration register.
```

Comment:
I think the docstring is not good enough, it does not say anything about velocity and does not explain what happens if some param is None.